src/chains.json

```diff
@@ -19,7 +19,7 @@
       "network": "mainnet",
       "networkId": 1,
       "nativeCurrency": {"name":"Ether","symbol":"ETH","decimals":18},
-      "web3": ["https://mainnet.infura.io/v3/891fe57328084fcca24912b662ad101f"],
```

Comment:
when we just change the api keys here - what will prevent the same thing happens again?

Reply:
It will be moved to .env. Still the tests are not working properly so I have to fix them first. :)